scripts/run_fuzzer.py

```diff
@@ -32,7 +32,7 @@
     elif param == '--no_checks':
         perform_checks = False
     elif param == '--enable_verification':
```

Comment:
`elif param.startswith('--enable_verification'):`

That should fix why enable_verification is always false